tests/zepkit/test/src/integration.test.js

```diff
@@ -18,9 +18,15 @@ function runIntegrationTest({ kit }, action) {
   before('cleaning up project folder', cleanup);
 
   it('unpack kit', function () {
-    run(`npx zos unpack ${kit}`);
+    // have to replace zos with local version so we test current build
+    run(`node ../../../packages/cli/lib/bin/zos-cli.js unpack ${kit}`);
   })
 
+  // have to replace zos with local version so we test current build
+  it('replace zos with local version', function () {
+    run('~/zos/node_modules/.bin/lerna bootstrap --scope=tests-cli-zepkit --scope="zos*"');
```

Comment:
`~/zos` will only work where the project is checked out directly on the home folder (for instance, it would fail on my machine). I'd change this for `npx lerna`.

Reply:
`npx lerna` fails on circleci. I'll figure it out. No worries.

Reply:
An alternative is to skip the lerna bootstrap step, run `npm install`, and then use [`npm link`](https://docs.npmjs.com/cli/link) manually to use the local one. If I understand correctly, you should run `npm link` on `packages/cli` to symlink it to the global scope, and then `npm link zos` in the workdir to link-install it.

Reply:
Yeah. I am aware of `npm link`. I'll go with lerna for now for the sake of simplicity and consistency.

Reply:
> npx lerna fails on circleci. I'll figure it out. No worries.

I wonder why it failed. Still, findUp is a good alternative!